src/logic/operate.js

```diff
@@ -0,0 +1,15 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  let result;
+  if (operation === '+') result = one + two;
+  if (operation === '-') result = one - two;
+  if (operation === 'x') result = one * two;
+  if (operation === '÷') result = one / two;
+  if (operation === '%') result = one * (two / 100);
+  return result;
+};
```

Comment:
[Optional]

Why not use the [`switch`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) statement or `else if` since `operation` can only be one of the tested symbols. By doing this other conditional won't need to run if a condition is passed already.

Also, let's handle division by 0. `Big` will throw an error that may crash the app in future milestones if not handled.

Reply:
Thank Bola, you're right, on the first implementation, I was returning the result inside the if blocks, I will put that back.